### PR TITLE
Adds a gib gun

### PR DESCRIPTION
**Pull request by Robertha89**
Adds a gibbing gun as an alternative to the russian revolver.
This is what it looks like:
image
This is what happens when you use it:
https://i.gyazo.com/69440e25eb2de2e1ead395b7b4471e19.gif

:cl: ppi
rscadd: Adds a gibbing gun.
/:cl:

### CODE REVIEW

**Comment by TullyBurnalot** [comment]
First time I post on the Git.

No.

**Comment by HereticChurch** [comment]
So, it's just a russian revolver that gibs you?
Why?

File: code/modules/projectiles/guns/special.dm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/obj/item/weapon/gun/gibgun
+	icon_state = "gibgun"
+	name = "gibbing gun"
+	desc = "A gun that gibs people."
+	fire_sound = 'sound/effects/gib.ogg'
+
+	afterattack(atom/target as mob|obj|turf, mob/living/user as mob|obj, flag, params)
```
Comment (Iamgoofball) [inline code/modules/projectiles/guns/special.dm]: # _**RELATIVE PATHING IS NOT ALLOWED**_

**Comment by Rumiluntti** [comment]
Wait.. why?

**Comment by Fox-McCloud** [comment]
It's not even like a Russian revolver; it just out and out gibs the person straight up.

:-1:

**Comment by Robertha89** [comment]
Considering this will only appear if either admins spawn it, or someone summons guns, not only that, it can function as a teaching tool to make higher levels of RP a thing, such as I found a gun on the floor, i should not shoot it wildly in the hallways. And it does not straight up gib em, is a little timer, but yeah, if you fired the gun, then you where intending to end anothers life, and karma is a bitch sometimes.

**Comment by Fox-McCloud** [comment]
half a second might as well be a "straight up" gib. 

This is still retarded as hell.

**Comment by Spacemanspark** [comment]
Maybe make it only work on yourself.

**Comment by Markolie** [comment]
Meh, if it's admin only why not. It's no different from the meteorite pen (instadeath) and has a nice sprite b

**Comment by Robertha89** [comment]
@Spacemanspark The gun only gibs the user, so the only one safe from it would be the clown, ironically.

**Comment by Plotron** [comment]
I don't think we need it...

**Comment by Spacemanspark** [comment]
Ah, I was tired when I looked at this.
I don't see why not, then, as long as it's adminspawnable only.

**Comment by TheDZD** [comment]
I honestly don't see the point of this.

**Comment by Robertha89** [comment]
Same reason as the chrono stuff, same reason as the l6 destroyer, same reason as the mjolnir, same reason as the gun vending machine, same reason for the three other reactive armors. 
I have stated possible uses, various reasons. It's basically a gun that punishes the user, it's a gun to kill griefers with, it's a teaching tool.

**Comment by TheDZD** [comment]
That makes absolutely no sense. Everything you mentioned is largely a tool for adminbus and admin events. This just gibs you if you use it, it's an adminbus version of a russian roulette revolver, which isn't needed due to the fact that admins can gib themselves via *many many* other methods. It's not really much of a teaching tool, considering griefers get banned anyways.

**Comment by taukausanake** [comment]
I think this is funny, but like everyone else said it's no good. If it was actually a Russian revolver and gave you that chance of not dying then I see why not. It's funny watching someone explode and cover everything nearby in viscera

**Comment by Fox-McCloud** [comment]
Doesn't seem to have support, and the suggestions here weren't implemented, either. Closing.

**Pull request closed without merging**